pwnlib/tubes/process.py

```diff
@@ -985,6 +985,50 @@ def leak(self, address, count=1):
             mem.seek(address)
             return mem.read(count) or None
 
+    readmem = leak
+
+    def writemem(self, address, data):
+        r"""Writes memory within the process at the specified address.
+
+        Arguments:
+            address(int): Address to write memory
+            data(bytes): Data to write to the address
+
+        Example:
+        
+            Let's write data to  the beginning of the mapped memory of the  ELF.
+
+            >>> context.clear(arch='i386')
+            >>> address = 0x100000
+            >>> data = cyclic(32)
+            >>> assembly = shellcraft.nop() * len(data)
+
+            Wait for one byte of input, then write the data to stdout
+
+            >>> assembly += shellcraft.read(0, 'esp', 1)
+            >>> assembly += shellcraft.write(1, address, 32)
+            >>> assembly += shellcraft.exit()
+            >>> asm(assembly)[32:]
+            b'1\xdb\x89\xe1j\x01Zj\x03X\xcd\x80j\x01[\xb9\xff\xff\xef\xff\xf7\xd1j Zj\x04X\xcd\x801\xdbj\x01X\xcd\x80'
+
+            Assemble the binary and test it
+
+            >>> elf = ELF.from_assembly(assembly, vma=address)
+            >>> io = elf.process()
+            >>> io.writemem(address, data)
```

Comment:
Python 3 write() returns the number of bytes written.
```suggestion
            >>> _ = io.writemem(address, data)
```